src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml

```diff
@@ -10,10 +10,10 @@
     d:DesignWidth="400">
     <StackPanel Orientation="Vertical">
         <ToolTipService.ToolTip>
-            <ToolTip>
+            <ToolTip Name="HotkeyTooltip">
                 <StackPanel Orientation="Vertical">
                     <TextBlock x:Uid="ShortcutWarningLabel"/>
-                    <TextBlock Text="{x:Bind Keys, Mode=OneTime}" FontWeight="SemiBold"/>
+                    <TextBlock Text="{x:Bind _keys, Mode=OneTime}" FontWeight="SemiBold"/>

```

Comment:
why do you bind to the private variable and not the public property as before?

Reply:
This is my first time programming these Xaml components, thank you in advance for helping me!

The reason I created the private field `_keys` was for creating a Setter that would update the visibility of the tooltip/tooltip icon.

Because all custom setters and getters were replicating the data to a private field, I believed that that was a "norm". Anyway, I will fix that ASAP!

Reply:
I will try to remove the private field and see how this goes!

Reply:
I tried not creating the private property and had a problem with infinite loops, just push a PR that uses the public field instead of the private one!